adapters/Pangle/PangleAdapter/Bidding/GADPangleRTBBannerRenderer.m

```diff
@@ -117,14 +111,7 @@ - (PAGBannerAdSize)bannerSizeFormGADAdSize:(GADAdSize)gadAdSize error:(NSError *
   } else if (size.height == kPAGBannerSize300x250.size.height) {
     return kPAGBannerSize300x250;
   }
-
-  if (error) {
-    *error = GADMAdapterPangleErrorWithCodeAndDescription(
-        GADPangleErrorBannerSizeMismatch,
-        [NSString stringWithFormat:@"Invalid size for Pangle mediation adapter. Size: %@",
-                                   NSStringFromGADAdSize(gadAdSize)]);
-  }
-  return (PAGBannerAdSize){CGSizeZero};
+  return kPAGBannerSize320x50;
```

Comment:
We shouldn't make this change. If the publisher requests a banner ad with 728x90, we shouldn't return one that is 320x50 but an error instead.